MovieQuiz/Services/QuestionFactory.swift

```diff
@@ -72,7 +82,8 @@ class QuestionFactory: QuestionFactoryProtocol {
             do {
                 imageData = try Data(contentsOf: movie.resizedImageURL)
             } catch {
-                print("Failed to load image")
+                self.delegate?.didFailToLoadData(with: QuestionFactoryError.loadImage)
```

Comment:
Тут надо проверить, не нужно ли обернуть на главную очередь:
```suggestion
                DispatchQueue.main.async { [weak self] in
                    self?.delegate?.didFailToLoadData(with: QuestionFactoryError.loadImage)
                }
```